databases/nosql/mongodb/README.md

```diff
@@ -8,7 +8,7 @@ A document-oriented database stores the data in so-called documents (pretty obvi
 
 In our sections on document-oriented databases we are assuming you have SQL knowledge. The following diagram should help identify the terms used in document-oriented databases to relate them to what you know in SQL (courtesy of www.beginnersbook.com):
 
-![rdb-mongo-pic](./assets//RDBMS_MongoDB_Mapping.jpg)
+![rdb-mongo-pic](./assets//RDBMS_MongoDB_Mapping.jpg) [ X ]
```

Comment:
This image works for me. Can you check again?
<img width="985" alt="image" src="https://github.com/HackYourFuture/study/assets/2033301/97bab9fe-6d9d-4ff0-8a1c-f060a76cd0dc">

Reply:
Yeh somehow it's working again.